Dockerfile.s3

```diff
@@ -0,0 +1,154 @@
+# base: Stage which installs necessary runtime dependencies (OS packages, java,...)
+FROM python:3.11.5-slim-buster@sha256:9f35f3a6420693c209c11bba63dcf103d88e47ebe0b205336b5168c122967edf as base
+ARG TARGETARCH
+
+# set workdir
+RUN mkdir -p /opt/code/localstack
+WORKDIR /opt/code/localstack/
+
+# Install runtime OS package dependencies
+RUN --mount=type=cache,target=/var/cache/apt \
+    apt-get update && \
+        # Install dependencies to add additional repos
+        apt-get install -y --no-install-recommends \
+            # Runtime packages (groff-base is necessary for AWS CLI help)
+             ca-certificates curl make openssl
+# do we need the dns server?
+#             ca-certificates curl make openssl iputils-ping iproute2
+# default under
+#             ca-certificates curl gnupg make openssl tar pixz zip unzip groff-base iputils-ping nss-passwords procps iproute2 xz-utils
```

Comment:
I wasn't sure if we should ship the DNS server, but maybe we should to enable virtual host. It shouldn't be too much work to enable it, but it might route everything to LS instead of just S3.